swaggertooling/openapi-pipeline/src/main/java/uk/gov/ch/pipe/NaiveValidatorPipe.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.ch.pipe;
+
+import uk.gov.ch.openapi.validator.OpenAPI3Validator;
+
+import java.io.IOException;
+
+public class NaiveValidatorPipe extends AbstractAPIPipe {
+    void validate(String... args) {
+        OpenAPI3Validator.main(args);
+    }
+
+    @Override
+    protected void handle() {
+        try {
+            validate(getArgs().getArgs("-o"));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Please log exception, not `printStackTrace()`

Reply:
getArgs not being used so no exception is thrown so no logging needed.